ground/gcs/src/libs/tlmapcontrol/src/mapwidget/tlmapwidget.cpp

```diff
@@ -369,7 +369,7 @@ namespace mapcontrol
     {
         internals::PointLatLng mcoord;
         bool reloc=false;
-        if(mcoord==internals::PointLatLng(0,0))
+        if(coord==internals::PointLatLng(0,0))
```

Comment:
yeah, I know, the code is really bad in those few files, spaces missing everywhere...